expr/expr.go

```diff
@@ -1073,14 +1073,19 @@ func EvalExpr(e *expr, from, until int32, values map[MetricRequest][]*MetricData
 			return nil, ErrMissingTimeseries
 		}
 
-		numerators, err := getSeriesArg(e.args[0], from, until, values)
+		var results []*MetricData
```

Comment:
Move this declaration nearer to where it's first used.  (For loop near line 1111 ?)